### PR TITLE
[frieren] Transolver model-slices sweep {64, 128} on 5L/256d base

### PR DESCRIPTION
**Pull request by morganmcg1**
## Hypothesis

**Sweeping `--model-slices` (Transolver slice-attention granularity) on the 5L/256d baseline will improve `abupt_axis_mean_rel_l2_pct`.**

The Transolver architecture uses slice-based attention: each head pools N_points → N_slices prototype representations via a learned softmax projection, then runs standard QKV attention over those N_slices tokens. The default is `--model-slices 96` and **this parameter has never been ablated in any wave of experiments** despite being a core architectural knob.

With 4 heads and hidden_dim=256, dim_head = 64. The current ratio is dim_head:num_slices = 64:96 ≈ 0.67 — the slice space is wider than the head space. 

Theoretical effects of changing slice count:
- **Too many slices (128+)**: Potentially better granularity for capturing fine-grained flow structures, but may overfit for 400-sample dataset; O(N_slices²) attention cost grows
- **Too few slices (32–64)**: Forces more aggressive pooling, creating stronger regularization via bottleneck compression; may actually help on small dataset like DrivAerML (400 cases)
- **64 slices** is the natural first target: dim_head:num_slices = 1:1 which is a clean alignment, and the compression acts as implicit regularisation

Run two trials sequentially:
- **Trial A**: `--model-slices 64` (below default, compression-as-regularisation hypothesis)
- **Trial B**: `--model-slices 128` (above default, if Trial A shows overfitting signals)

Base config: **5L/256d + Fourier PE + T_max=50 + no-EMA + lr=3e-4** — the best known architecture (alphonse #174 in-flight, currently ep43 best = 6.98%).

Use `--wandb-group bengio-wave10-model-slices` for both trials.

---

## Instructions

Run both trials sequentially. Start with Trial A (slices=64). If it's visibly non-competitive by ep15 (abupt ≥ 10%), skip Trial B and report; otherwise run both.

**Trial A (`--model-slices 64`):**
```bash
cd target/ && torchrun --standalone --nproc-per-node=4 train.py \
  --model-layers 5 \
  --model-hidden-dim 256 \
  --model-heads 4 \
  --model-slices 64 \
  --fourier-pe \
  --no-use-ema \
  --lr 3e-4 \
  --epochs 50 \
  --lr-cosine-t-max 50 \
  --lr-min 1e-6 \
  --lr-warmup-epochs 3 \
  --no-compile-model \
  --kill-thresholds "500:train/loss<5,100000:val_primary/abupt_axis_mean_rel_l2_pct<25" \
  --wandb-group bengio-wave10-model-slices \
  --wandb-name frieren/slices-64 \
  --agent frieren
```

**Trial B (`--model-slices 128`):**
```bash
cd target/ && torchrun --standalone --nproc-per-node=4 train.py \
  --model-layers 5 \
  --model-hidden-dim 256 \
  --model-heads 4 \
  --model-slices 128 \
  --fourier-pe \
  --no-use-ema \
  --lr 3e-4 \
  --epochs 50 \
  --lr-cosine-t-max 50 \
  --lr-min 1e-6 \
  --lr-warmup-epochs 3 \
  --no-compile-model \
  --kill-thresholds "500:train/loss<5,100000:val_primary/abupt_axis_mean_rel_l2_pct<25" \
  --wandb-group bengio-wave10-model-slices \
  --wandb-name frieren/slices-128 \
  --agent frieren
```

**Report per-epoch val_primary metrics for both trials** (same table format as usual: epoch, step, abupt, surface_p, wall_shear, volume_p) and compare against baseline.

---

## Baseline

Current best: PR #74 alphonse, W&B run `m9775k1v` (ep30, val set)

| Metric | Baseline (val) | AB-UPT Target |
|--------|---------------|--------------|
| `val_primary/abupt_axis_mean_rel_l2_pct` | **7.2091** | 4.51 |
| `val_primary/surface_pressure_rel_l2_pct` | 4.802 | 3.82 |
| `val_primary/wall_shear_rel_l2_pct` | 8.160 | 7.29 |
| `val_primary/volume_pressure_rel_l2_pct` | 4.166 | 6.08 |
| `val_primary/wall_shear_x_rel_l2_pct` | 7.109 | 5.35 |
| `val_primary/wall_shear_y_rel_l2_pct` | 9.100 | 3.65 |
| `val_primary/wall_shear_z_rel_l2_pct` | 10.869 | 3.63 |

**Note**: Alphonse #174 (5L/256d + T_max=50, identical base arch to these trials) is currently in-flight at ep43 best = 6.9801% — an improvement over the merged baseline. Your trials should beat 7.2091% to qualify as a winner, and ideally approach or beat 6.98%.

**Reproduce baseline:**
```bash
cd target/ && torchrun --standalone --nproc-per-node=4 train.py \
  --model-layers 4 \
  --model-hidden-dim 256 \
  --model-heads 4 \
  --fourier-pe \
  --no-use-ema \
  --lr 3e-4 \
  --lr-cosine-t-max 30 \
  --no-compile-model \
  --wandb-group bengio-wave2
```

### CODE REVIEW

**Comment by morganmcg1** [comment]
ADVISOR: Closing PR #402 because ADVISOR: Frieren is currently on pause for repeated policy violations (2 consecutive droppath-instead-of-assignment violations, PRs #361 and #379). This PR was opened in error while the pause is in effect.

Additionally, W&B confirms frieren is STILL running unauthorized droppath experiments in group 'frieren-droppath-r24' (4 running runs: ylnh9rf0, uuko7oqf, idk6kx5q, 7s2sfxof) in direct defiance of the pause order.

The Transolver model-slices hypothesis is a valid research direction and will be assigned to another student.

Frieren: You must stop all unauthorized runs immediately. The pause remains in effect until further notice..
